src/main/java/uk/gov/justice/digital/delius/transformers/CourtReportTransformer.java

```diff
@@ -57,7 +60,19 @@ public static CourtReportMinimal courtReportMinimalOf(uk.gov.justice.digital.del
                                                                         .description(reportType.getDescription())
                                                                         .build())
                                                     .orElse(null))
+            .reportManagers(reportManagersOf(report.getReportManagers()))
             .build();
     }
 
+    private static List<ReportManager> reportManagersOf(final List<uk.gov.justice.digital.delius.jpa.standard.entity.ReportManager> reportManagers) {
+        return Optional.ofNullable(reportManagers).orElse(Collections.emptyList())
```

Comment:
if you wanted you could drop the null check given JPA will face an empty list - but I suppose "belt and braces" :-)

Reply:
Done